apps/codeforafrica/src/lib/data/utils/posts.js

```diff
@@ -0,0 +1,185 @@
+import formatDate from "@/codeforafrica/utils/formatDate";
+
+export function formatTags(tags) {
+  const excludedTags = new Set(["stories", "opportunities"]);
+
+  const tagCounts = tags.reduce((counts, { name }) => {
```

Comment:
Remove exclude tags from this point so that we don't need to worry about them when sorting.